otelib/transformation.py

```diff
@@ -0,0 +1,42 @@
+import json
+import requests
+
+from oteapi.models import TransformationConfig
+
+from .apierror import ApiError
+from .abstractfilter import AbstractFilter
+from .settings import Settings
```

Comment:
```suggestion
from oteapi.apierror import ApiError
from oteapi.abstractfilter import AbstractFilter
from oteapi.settings import Settings
```

Reply:
Hmm, I am still not convinced... Both absolute and relative imports will break if you move files around. I see no difference in robustness in that respect. But I agree that my argument that the absolute path may lead to conflicts with other packages is not a good argument, since you are specifying the full path from the root package. However, that has the effect that `oteapi` must be properly installed on your system before you can test it. Relative imports does not have this requirement.